packages/cli/src/handlers/generate.ts

```diff
@@ -58,25 +61,13 @@ export const generateHandler = async (options: GenerateHandlerOptions) => {
 
     const absoluteProjectPath = path.resolve(options.projectDir);
     const absoluteProfilesPath = path.resolve(options.profilesDir);
-
-    const context = await getDbtContext({
-        projectDir: absoluteProjectPath,
-        verbose: options.verbose,
-    });
+    const context = await getDbtContext({ projectDir: absoluteProjectPath });
     const profileName = options.profile || context.profileName;
-    if (options.verbose)
```

Comment:
Do we want to remove this or was it a bad merge conflict?

Reply:
good catch, it was a conflict. Fixed.